sormas-ui/src/main/java/de/symeda/sormas/ui/campaign/campaigndata/CampaignFormDataEditForm.java

```diff
@@ -113,6 +115,21 @@ protected void addFields() {
 				}
 			});
 		}
+
+		final UserDto currentUser = UserProvider.getCurrent().getUser();
+
+		if (currentUser.getRegion() != null) {
+			cbRegion.setValue(currentUser.getRegion());
+			cbRegion.setReadOnly(true);
```

Comment:
use setEnabled(false) instead, read only leads to the field disappearing and only the caption being displayed which looks weird